### ISSUE TITLE
[css-multicol] editorial: <line-width> tooltip is incomplete

### ISSUE THREAD

**Issue by MatsPalmgren** [w3c/csswg-drafts#6559]
https://drafts.csswg.org/css-multicol/#crw
Hover the `<line-width>` link - a tooltip appears that says "Expands to: medium | thick | thin".
This is misleading since it omits `<length [0,∞]>` (which you will see if you click the link).

This might be an issue that affects other specs too...

CC @tabatkins

**Comment by tabatkins** [comment]
Yeah it probably does affect a bunch. Terminals are expanded like that from dfns listed as `for="<that-terminal>"`; following the `<line-width>` length, I see that only the keywords are actually dfn'd for it.

Since Reffy is collecting grammars, I can probably switch this feature over to using that data; it'll presumably be more complete. I should investigate this.
